Formula/nwchem.rb

```diff
@@ -17,6 +16,7 @@ class Nwchem < Formula
   depends_on "gcc" # for gfortran
   depends_on "open-mpi"
   depends_on "openblas"
+  depends_on "python"
```

Comment:
Can this work with python@3.8?

Reply:
Yes, it does. I have tested it on linux. I have tested under the current python brew 3.7.
Could you please tell me what change are you suggesting?

Reply:
@SMillerDev Are you suggesting that the assignment of PYTHONVERSION should be flexible to automatically detect the python version?
Would the following code work
```
pymajor = `python3 -c 'import sys;print(sys.version_info.major)'`.chomp
pyminor = `python3 -c 'import sys;print(sys.version_info.minor)'`.chomp
ENV["PYTHONVERSION"] = pymajor + '.' + pyminor
```

Reply:
Pushed commit https://github.com/Homebrew/homebrew-core/pull/50745/commits/7c5b5e172ed96852ba0d555ee2fa05afe1a13b67 to automatically detect PYTHONVERSION

Reply:
No, he means using `depends_on "python@3.8"`.

Reply:
@Bo98 and @SMillerDev , I prefer the last commit that detects the python3 available, instead of asking for a specific python version

Reply:
@edoapra You should still change this to 
```suggestion
  depends_on "python@3.8"
```

If it will work. The reasoning for this is that Python 3.8 has broken a lot of packages and we've split it out as a separate version as we migrate `python` from 3.7 to 3.8. So anything that can `depends_on python@3.8` should and we will automatically go through and change them all to `depends_on python` once the update/migrations is finished.

Reply:
This change will not trigger the use of python@3.8 in NWChem, because of the automatic detection of python3. Do you want me the following line to add that enforces use of 3.8, too?
```
ENV["PYTHONVERSION"] = 3.8
```